lib/tasks/surveys.rake

```diff
@@ -94,8 +94,11 @@ namespace :surveys do
 
   desc 'Find SurveyNotifications that haven\'t been sent and send them'
   task send_notifications: :environment do
-    SurveyNotification.active.each do |notification|
-      notification.send_email
-    end
+    SurveyNotification.active.collect(&:send_email)
+  end
+
+  desc 'Send follow-up survey notifications if configured on the parent notification'
+  task send_survey_notification_follow_ups: :environment do
+    SurveyNotification.active.collect(&:send_follow_up)
```

Comment:
Yeah, it should be `each`.

On Wed, May 4, 2016 at 11:56 AM, Sage Ross <notifications@github.com> wrote:

> In lib/tasks/surveys.rake
> <https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/843#discussion_r62094891>
> :
>
> > @@ -94,8 +94,11 @@ namespace :surveys do
> >
> >    desc 'Find SurveyNotifications that haven\'t been sent and send them'
> >    task send_notifications: :environment do
> > -    SurveyNotification.active.each do |notification|
> > -      notification.send_email
> > -    end
> > +    SurveyNotification.active.collect(&:send_email)
> > +  end
> > +
> > +  desc 'Send follow-up survey notifications if configured on the parent notification'
> > +  task send_survey_notification_follow_ups: :environment do
> > +    SurveyNotification.active.collect(&:send_follow_up)
>
> Why collect instead of each?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/843/files/1aea670cfe086046833f0728c2e69d2ba85be33e..49d5a3b3970830afb541a3bd7613ff4ef929388c#r62094891>
>



-- 
*W* *I* *N* *T* *R*  |  Nick Cox |  Lead Engineer |  o. 206.783.4570  |
wintr.us